fitbenchmarking/results_processing/base_table.py

```diff
@@ -217,11 +217,19 @@ def to_html(self, table, colour, links, error):
         :return: HTLM table output
         :rtype: str
         """
+        link_template = '<a href="https://fitbenchmarking.readthedocs.io/en/'\
+            'latest/concept/how/Minimizers.html#{0}">{0}</a>'
```

Comment:
```suggestion
            'latest/concept/how/Minimizers.html#{0}" target="_blank">{0}</a>'
```

Reply:
I think this should open the docs in a new tab, which might be better than navigating away from the results page completely.

Reply:
Yea I like that better. I have created a PR to fix the `sasview`/`bumps` naming issue so once that goes in then the naming conversion should work with the links